File: patch.js
const fs = require('fs');
let code = fs.readFileSync('src/components/chat-cards/FoodCard.tsx', 'utf8');

const regex = /{activeScoutItems\.some\(\(i: any\) => \(i\.nutritionFacts && Object\.keys\(i\.nutritionFacts\)\.length > 0\) \|\| \(i\.rawNutritionLabel && Object\.keys\(i\.rawNutritionLabel\)\.length > 0\)\) && \([\s\S]*?<\/div>\n                             \)}/m;

code = code.replace(regex, '<NutritionLabelTable activeScoutItems={activeScoutItems} />');
fs.writeFileSync('src/components/chat-cards/FoodCard.tsx', code);

File: patch.py
import re

with open('src/components/chat-cards/FoodCard.tsx', 'r') as f:
    code = f.read()

pattern = r"\{activeScoutItems\.some\(\(i: any\) => \(i\.nutritionFacts && Object\.keys\(i\.nutritionFacts\)\.length > 0\) \|\| \(i\.rawNutritionLabel && Object\.keys\(i\.rawNutritionLabel\)\.length > 0\)\) && \([\s\S]*?<\/div>\n                             \)}"

new_code = re.sub(pattern, "<NutritionLabelTable activeScoutItems={activeScoutItems} />", code)

with open('src/components/chat-cards/FoodCard.tsx', 'w') as f:
    f.write(new_code)
